tests/system/feature_store/test_feature_store.py

```diff
@@ -406,20 +406,22 @@ def test_get_offline_features_with_or_without_indexes(self):
         ), "'another_time' column should be present"
 
         # with_indexes = False, entity_timestamp_column = "invalid" - should return the timestamp column
-        df_with_time = fstore.get_offline_features(
-            vector, entity_timestamp_column="another_time"
-        ).to_dataframe()
+        df_without_time_and_indexes = fstore.get_offline_features(vector).to_dataframe()
         assert isinstance(
-            df_with_time.index, pd.core.indexes.range.RangeIndex
+            df_without_time_and_indexes.index, pd.core.indexes.range.RangeIndex
         ), "index column is not of default type"
-        assert df_with_time.index.name is None, "index column is not of default type"
         assert (
-            "ticker" not in df_with_time.columns
+            df_without_time_and_indexes.index.name is None
+        ), "index column is not of default type"
+        assert (
+            "ticker" not in df_without_time_and_indexes.columns
         ), "'ticker' column shouldn't be present"
-        assert "time" in df_with_time.columns, "'time' column should be present"
         assert (
-            "another_time" not in df_with_time.columns
-        ), "'another_time' column should not be present"
+            "time" not in df_without_time_and_indexes.columns
+        ), "'time' column should be present"
```

Comment:
Hm, you added a `not`, so now the message is wrong. If the new behaviour is correct, feel free to delete the unnecessary message (or otherwise, need to fix it).